packages/api/src/routes/ehr/athenahealth/chart.ts

```diff
@@ -40,4 +39,62 @@ router.post(
   })
 );
 
+/**
+ * POST /ehr/athenahealth/chart/:id/condition
+ *
+ * Tries to write the condition to the patient's chart
+ * @param req.params.id The ID of AthenaHealth Patient.
+ * @param req.body The FHIR Resource payload
+ * @returns Athena API response
+ */
+router.post(
+  "/:id/condition",
+  handleParams,
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getCxIdOrFail(req);
+    const athenaPatientId = getFrom("params").orFail("id", req);
+    const athenaPracticeId = getFromQueryOrFail("practiceId", req);
+    const athenaDepartmentId = getFromQueryOrFail("departmentId", req);
+    const payload = req.body; // TODO Parse body https://github.com/metriport/metriport-internal/issues/2170
+    const conditionDetails = await writeConditionToChart({
+      cxId,
+      athenaPatientId,
+      athenaPracticeId,
+      athenaDepartmentId,
+      condition: payload,
+    });
+    return res.status(httpStatus.OK).json(conditionDetails);
```

Comment:
These endpoints are ever only used by the Athena EHR UI/FE? Just because we're tying the response from our endpoint to the response from Athena - if they change something, it will directly impact the result of our API.

Reply:
It's only used by the FE, and even then it's response is not used at all anyway (yet).